tests/integration/src/test/java/org/mule/test/integration/ServerTools.java

```diff
@@ -1,187 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
- */
-package org.mule.test.integration;
-
-import org.mule.transport.file.filters.FilenameWildcardFilter;
-import org.mule.util.FileUtils;
-
-import java.io.File;
-
-import javax.jms.JMSException;
-
-import org.apache.activemq.ActiveMQConnection;
-import org.apache.activemq.ActiveMQConnectionFactory;
-import org.apache.tools.ant.BuildException;
-import org.apache.tools.ant.DefaultLogger;
-import org.apache.tools.ant.Project;
-import org.apache.tools.ant.taskdefs.ExecuteWatchdog;
-import org.apache.tools.ant.taskdefs.Java;
-import org.apache.tools.ant.types.Environment;
-import org.apache.tools.ant.types.Path;
-import org.apache.tools.ant.util.Watchdog;
-
-/**
- * Will start external test servers needed for the integration tests
- */
-
-public class ServerTools
```

Comment:
looks like this can be useful for JMS testing and stuff. Why do you include it in this removal?